apps/home-dashboard/src/App.css

```diff
@@ -37,5 +40,5 @@
 }
 
 .read-the-docs {
-  color: #888;
+  color: #888888;
```

Comment:
this has been changed automatically by stylelint... it standardizes all the hex to 6 digits. My OCD is happy